File: Python/problem015.py
#! /usr/bin/env python3
"""
Problem 15 - Project Euler
http://projecteuler.net/index.php?section=problems&id=015
"""
from math import factorial

if __name__ == "__main__":
    # calc 40_C_20
    m = 40
    n = 20
    print(int(factorial(m) / (factorial(m - n) * factorial(n))))

File: Clojure/problem015.clj
#! /usr/bin/env clojure
; Problem 15 - Project Euler
; http://projecteuler.net/index.php?section=problems&id=015

(defn factorial [x]
  (reduce *' (range 1 (inc x))))

(let [m 40
      n 20]
  (println (long
    (/ (/ (factorial m) (factorial (- m n)))
       (factorial n)))))
